packages/dids/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@tbd54566975/dids",
```

Comment:
good question @KendallWeihe was wondering the same myself. i called it `dids` because there are many did method implementations.

any thoughts @frankhinek ?